aspnet/conceptual-overview/aspnet.rst

```diff
@@ -1,37 +1,37 @@
+The first preview release of ASP.NET came out almost 15 years ago as part of the .NET Framework.  Since then millions of developers have used it to build and run great web apps, and over the years we have added and evolved many capabilities to it.
 
-With ASP.NET Core we are making a number of architectural changes that make the core web framework much leaner and more modular. ASP.NET Core is no longer based on System.Web.dll, but is instead based on a set of granular and well factored NuGet packages allowing you to optimize your app to have just what you need. You can reduce the surface area of your application to improve security, reduce your servicing burden and also to improve performance in a true pay-for-what-you-use model.
+ASP.NET Core has a number of architectural changes that result in a much leaner and modular framework. ASP.NET Core is no longer based on *System.Web.dll*, but is instead based on a set of granular and well factored `NuGet <http://www.nuget.org/>`__ packages allowing you to optimize your app to include just the NuGet packages you need. You can reduce your apps surface area to improve security, reduce  servicing and improve performance in a pay-for-what-you-use model.
 
-ASP.NET Core is built with the needs of modern Web applications in mind, including a unified story for building Web UI and Web APIs that integrate with today's modern client-side frameworks and development workflows. ASP.NET Core is also built to be cloud-ready by introducing environment-based configuration and by providing built-in dependency injection support.
+ASP.NET Core is built with the needs of modern Web apps in mind, including a unified story for building Web UI and Web APIs that integrate with today's modern client-side frameworks and development workflows. ASP.NET Core is built to be cloud-ready by introducing environment-based configuration and by providing built-in :doc:`dependency injection </fundamentals/dependency-injection>` support.
 
-To appeal to a broader audience of developers, ASP.NET Core supports cross-platform development on Windows, Mac and Linux. The entire ASP.NET Core stack is open source and encourages community contributions and engagement. ASP.NET Core comes with a new, agile project system in Visual Studio while also providing a complete command-line interface so that you can develop using the tools of your choice.
+To appeal to a broader audience of developers, ASP.NET Core supports cross-platform development on Windows, Mac and Linux. The entire ASP.NET Core stack is open source and encourages community contributions and engagement. ASP.NET Core comes with a new, agile project system in Visual Studio while also providing a complete command-line interface so that you can develop using the tools of your choice. `Visual Studio Code <https://code.visualstudio.com/#alt-downloads>`__ runs on a variety of platforms.
```

Comment:
**Rephrase third sentence.** ASP.NET Core comes with a new, agile project system in Visual Studio and a complete command-line interface, so that you can develop using the tools of your choice.